autogpt_platform/backend/backend/blocks/basic.py

```diff
@@ -3,13 +3,50 @@
 
 from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema, BlockType
 from backend.data.model import SchemaField
+from backend.util.file import store_temp_file
 from backend.util.mock import MockObject
 from backend.util.text import TextFormatter
 from backend.util.type import convert
 
 formatter = TextFormatter()
 
 
+class FileStoreBlock(Block):
+    class Input(BlockSchema):
+        file_in: str = SchemaField(
+            description="The file to store in the temporary directory, it can be a URL, data URI, or local path."
+        )
```

Comment:
For UX it could make sense to split this into a `DownloadFileBlock` and `StoreFileBlock`.

Since this is for *temporary* files though, `StoreFileBlock` should probably be reserved for something that stores to persistent user file storage. This block can be named `MakeTemporaryFileBlock` or something like that.

Reply:
This is equivalent to `ValueStoreBlock`, why can't we make this block allow both permanent and temporary storage?